cli/dts/lib.deno.window.d.ts

```diff
@@ -22,4 +25,25 @@ declare const self: Window & typeof globalThis;
 declare const onload: ((this: Window, ev: Event) => any) | null;
 declare const onunload: ((this: Window, ev: Event) => any) | null;
 
+/**
+ * Shows the given message and waits for the enter key pressed.
+ * @param message
+ */
+declare function alert(message?: string): void;
+
+/**
+ * Shows the given message and waits for the answer. Returns the user's answer as boolean.
+ * @param message
+ */
+declare function confirm(message?: string): boolean;
+
+/**
+ * Shows the given message and waits for the user's input. Returns the user's input as string.
+ * If the default value is given and the user inputs the empty string, then it returns the given
```

Comment:
I think this should also document when prompt can return null

Reply:
I agree with that, CC @kt3k